packages/realm_dart/lib/src/native/realm_bindings.dart

```diff
@@ -3999,21 +3999,23 @@ class RealmLibrary {
     ffi.Pointer<ffi.Void> userdata,
     int transferred_bytes,
     int total_bytes,
+    double estimate,
```

Comment:
I know this is not part of this PR per se, but just a side effect of updating realm-core, but what is the relation between these 3 args. I assume it is not `estimate = transferred_bytes / total_bytes`.